resources/js/components/textarea-autosize.tsx

```diff
@@ -24,7 +24,7 @@ export default class TextareaAutosize extends React.PureComponent<Props, State>
   private shouldUpdate = true;
 
   private get maxHeight() {
-    return this.props.maxRows != null && this.state.lineHeight != null
+    return this.props.maxRows != null && this.state.lineHeight != null && Number.isFinite(this.state.lineHeight)
```

Comment:
not really important but if lineHeight is typed non nullable and initialized with NaN, only isFinite check is needed